docker-compose.yml

```
@@ -34,11 +34,13 @@ services:
     environment:
       - DEBUG=${DEBUG-}
       - DOCKER_HOST=unix:///var/run/docker.sock
+      - AWS_DEFAULT_REGION=${AWS_REGION:-us-west-2}
     networks:
       - app
     volumes:
       - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
       - "/var/run/docker.sock:/var/run/docker.sock"
+      - "${PWD}/localstack/entrypoint/init-aws.sh:/etc/localstack/init/ready.d/init-aws.sh"
```

I think leaving out `${PWD}` and using a relative path should be basically equivalent, but avoid potential ambiguity/issues from running `docker compose` from another directory.

```suggestion
      - "localstack/entrypoint/init-aws.sh:/etc/localstack/init/ready.d/init-aws.sh"
```

---

Sounds good!